cmd/manager/operator.go

```diff
@@ -241,6 +249,45 @@ func ensureDefaultProfileBundles(ctx context.Context, crclient client.Client, na
 	return lastErr
 }
 
+func ensureDefaultScanSettings(ctx context.Context, crclient client.Client, namespaceList []string) error {
+	var lastErr error
+	for _, ns := range namespaceList {
+		pb := &compv1alpha1.ScanSetting{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      defaultScanSettingsName,
+				Namespace: ns,
+			},
+			ComplianceScanSettings: compv1alpha1.ComplianceScanSettings{
+				Debug: false,
+				RawResultStorage: compv1alpha1.RawResultStorageSettings{
```

Comment:
I like the general idea of creating the default object, but do we need to explicitly mention the defaults? It appears that the defaults are applied by the respective controllers when the objects are already created. And should we ever change the defaults, the example object wouldn't change unless we also update it, unlike the defaults applied when creating the CRs.

Reply:
We probably don't need to specify defaults for the default object. Let's do this, I'll only leave the defaults that come from a constant.